manifests/init.pp

```diff
@@ -118,8 +109,27 @@
     owner   => 'root',
     group   => 'root',
     mode    => '0640',
-    content => "OPTIONS=\"-A -u ntp:ntp -p /var/run/ntpd.pid\"
-SYNC_HWCLOCK=yes\n",
+    content => "OPTIONS=\"-A -u ntp:ntp -p /var/run/ntpd.pid\"\n",
+    notify  => Service['ntpd']
+  }
+  file { '/etc/sysconfig/ntpdate':
+    ensure  => 'file',
+    owner   => 'root',
+    group   => 'root',
+    mode    => '0640',
+    content => "SYNC_HWCLOCK=yes\n",
+    notify  => Service['ntpd']
+  }
+
+  if $servers =~ Array {
+    $_servers = $servers
+  }
+  else {
+    $_servers = $servers.keys
+  }
+  file { '/etc/ntp/step-tickers':
+    ensure  => 'file',
+    content => epp("${module_name}/step-tickers.epp", { 'ntp_servers' => $_servers }),
```

Comment:
is there any reason in which the step-tickers could be different than the NTP servers?

Reply:
No, but I can make a param for it